EIPS/eip-draft_location-metadata-standard.md

```diff
@@ -0,0 +1,189 @@
+---
+eip: <to be assigned>
+title: Location Metadata Standard in ERC-721 Tokens
+author:
+Ricardo Chacon (@RicardoChacon)
+Isabella Costanza (@isabellacostanza)
+Gabriela Garro (@gabygarro)
+James Jose (@jamesjosegdsey)
+David Mendes (@de-mendes)
+discussions-to: https://ethereum-magicians.org/t/request-for-feedback-standardizing-location-metadata-and-other-metadata-in-erc-721-tokens/5985
+status: Draft
+type: Informational
+created: 2021-05-04
+requires: 721
+---
+
+## Simple Summary
+
+Standard for location metadata in the ERC-721 tokens. Tokens wishing to include physical location metadata should follow the same standard.
+
+## Abstract
+
+This standard specifies how location metadata should be represented in an ERC-721 token. It serves as an extension of the ERC-721 Metadata Standard as it adds a "location" field to the JSON schema. This location metadata can be represented with either the physical address, geolocation coordinates or both for maximum clarity. 
+
+## Motivation
+
+This standard will promote composability and cohesiveness of tokens across across platforms and systems. Use cases which rely on location metadata or that would benefit from implementing it often find themselves in the position of structuring this data in whichever way serves their specific purpose. However, representing location data is not a trivial endeavor and too many implementations exist since each one is use case specific, even though they are mostly similar. Therefore, we have leveraged existing ISO and geolocation standards to develop a metadata structure that can be applied globally.
+
+Moreover, inconsistent data formats present a rather challenging situation if an user wants to make use of this data, as it can be computationally expensive and slow to clean it up. This standard provides a solution by setting a standard which will cover the need of having location metadata and also give the community a framework which will prove tremendously useful at the application layer.
+
+One example where consistent location data structures are crucial is in supply chain management. As business partners try to coordinate their inventory to eliminate the potential for stockouts or spoilage, they need to understand the location of tokenized inputs. Tracking these inputs on one interface is only possible if all tokens are using the same standards for recording location. 
+
+## Specification
+
```

Comment:
why not to add support for a geo uri representation

Reply:
We changed the design to incorporate Schema.org which will allow to add aGeo URI field as part of the identifier field in the Place Schema from Schema.org